budget_proj/bin/env-template.sh

```diff
@@ -0,0 +1,25 @@
+#! /bin/bash
+
+#
+# To connect to database, make a copy of this file as "env.sh" and replace VALUES as noted
+# 
```

Comment:
Suggested addition to the documentation:
# Then source the file in the shell where you will build or run the server. For example:
#      $ source ./build_proj/bin/env.sh